tests/test_basic.py

```diff
@@ -297,3 +297,71 @@ def test_mons_up_down(rook_cluster):
         check += 1
 
     assert mon_pods == mons
+
+
+def test_rbd_pvc(rook_cluster):
+    # create a CephBlockPool
+    output = rook_cluster.kubernetes.kubectl_apply(
+                os.path.join(
+                    rook_cluster.ceph_dir, 'csi/rbd/storageclass.yaml'))
+
+    if output[0] != 0:
+        pytest.fail("Could not create a CephBlockPool StorageClass")
+
+    # check if StorageClass is up and available
+    output = rook_cluster.kubernetes.kubectl('get sc rook-ceph-block')
+
+    if output[0] != 0:
+        pytest.fail("Could not find a CephBlockPool called StorageClass")
+
+    # create an rbd based PVC
+    output = rook_cluster.kubernetes.kubectl_apply(
+                os.path.join(rook_cluster.ceph_dir, 'csi/rbd/pvc.yaml'))
+
+    if output[0] != 0:
+        pytest.fail("Could not create a rbd-PVC")
+
+    pvc = rook_cluster.kubernetes.kubectl('get -o yaml pvc rbd-pvc')
+
+    if pvc[0] != 0:
+        pytest.fail("Could not get rbd-pvc details")
+
+    pvc_object = yaml.load(pvc[1], Loader=yaml.BaseLoader)
+
+    i = 0
+    while pvc_object['status']['phase'] != 'Bound':
+        if i == 6:
+            pytest.fail("PVC rbd-pvc ist still not bound")
+            break
+        time.sleep(10)
+        pvc = rook_cluster.kubernetes.kubectl('get -o yaml pvc rbd-pvc')
+        if pvc[0] != 0:
+            pytest.fail("Could not get rbd-pvc details")
+            break
+        pvc_object = yaml.load(pvc[1], Loader=yaml.BaseLoader)
+        i += 1
+
+    # create a pod using the PVC
+    output = rook_cluster.kubernetes.kubectl_apply(
+                os.path.join(rook_cluster.ceph_dir, 'csi/rbd/pod.yaml'))
+
+    if output[0] != 0:
+        pytest.fail("Could not create a rbd-pod")
+
+    pod = rook_cluster.kubernetes.kubectl('get -o yaml pod csirbd-demo-pod')
+
+    if pvc[0] != 0:
+        pytest.fail("Could not get csirbd-demo-pod details")
+
+    pod_object = yaml.load(pod[1], Loader=yaml.BaseLoader)
+
+    i = 0
+    while pod_object['status']['phase'] != 'Running':
+        if i == 10:
+            pytest.fail("Pod csirbd-demo-pod did not start")
+            break
+        time.sleep(10)
+        pod = rook_cluster.kubernetes.kubectl(
+            'get -o yaml pod csirbd-demo-pod')
+        pod_object = yaml.load(pod[1], Loader=yaml.BaseLoader)
+        i += 1
```

Comment:
This test leaves all of its resources set up. It should probably tear them down?

Reply:
You are correct.
I am using `wait_for_result instead` now.